src/fondant/cli.py

```diff
@@ -607,18 +607,28 @@ def run_local(args):
     from fondant.dataset.runner import DockerRunner
 
     extra_volumes = []
+    # use workspace from cli command
+    # if args.workspace exists
+
+    workspace = getattr(args, "workspace", None)
```

Comment:
I've added already the workspace to the cli. For now, the implementation is using a default workspace. 
The run command loads the dataset object of the file (former pipeline) and executes the operations. 

As part of #887  I would handle a proper default workspace initialisation, and we can think about refer to a workspace file (for instance .workspace/local.env containing information of environment and runner) or a definition inside the source code. 
At least something like this `fondant run local ... --workspace ./local.env` feels natural.